include/deal.II/lac/block_linear_operator.h

```diff
@@ -171,7 +171,8 @@ block_back_substitution(
  * auto top_left_op = linear_operator(top_left);
  * auto bottom_right_op = linear_operator(bottom_right);
  * std::array<decltype(top_left_op), 2> operators {{top_left_op,
- * bottom_right_op}}; auto block_op = block_diagonal_operator (operators);
+ *                                                  bottom_right_op}};
```

Comment:
sorry for not catching this before, but it looks like we truncated the `std::array` above.

Reply:
Hmm... Can you be more specific? I don't quite get the problem.

Reply:
No, I misread the diff. This is fine. sorry